src/components/ProgressBar/ProgressBar.scss

```diff
@@ -40,11 +40,29 @@
   height: progress-bar-height(large);
 }
 
+.colorHighlight {
+  --p-progressbar-background: var(--p-surface-neutral, #{color('sky')});
+  --p-progressbar-indicator: var(--p-border-highlight, #{color('teal')});
+}
```

Comment:
Not sure component-specific custom properties are the right way to fix this, guidance would be appreciated.